ros2_control_test_nodes/test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
we have linters for this on the CI, let's use those please, I'd really prefer if we could get rid of junk like this